site/content/docs/5.0/examples/floating-labels/floating-labels.css

```diff
@@ -76,7 +76,8 @@ body {
   color: #777;
 }
 
-.form-label-group input:not(:placeholder-shown) ~ label {
+.form-label-group input:not(:placeholder-shown) ~ label,
+.form-label-group input:-webkit-autofill ~ label {
```

Comment:
Since [`:-webkit-autofill` isn't supported](https://caniuse.com/?search=%3A-webkit-autofill) in all [our supported browsers](https://github.com/twbs/bootstrap/blob/main/.browserslistrc) (Firefox and Edge 16 <> 79), this selector invalidates the whole declaration block for some of them.

There's [an internal pseudo-class in Firefox (`:-moz-auto-fill`)](https://bugzilla.mozilla.org/show_bug.cgi?id=1355438) but it doesn't seem to be available for authors as of now.

So in order to apply this as a progressive enhancement, we need to dedupe those selectors (and duplicate the rules).

Reply:
Does the current autoprefixer version support this property? I haven't tried it yet.

Reply:
Hi @XhmikosR, where can I learn on how to check this info?

Reply:
https://github.com/postcss/autoprefixer/issues/626 is outdated by a few years, unsure if there's anything else for it.

Reply:
note that Firefox doesn't have this bug in the first place, so autoprefixing may not be an issue here (for now anyway, unless it turns out Chrome are doing this on purpose as a security measure, and Firefox later copies the behavior)